packages/eslint-plugin/package.json

```diff
@@ -46,18 +46,18 @@
   "dependencies": {
     "@typescript-eslint/experimental-utils": "4.31.0",
     "@typescript-eslint/scope-manager": "4.31.0",
-    "debug": "^4.3.1",
+    "debug": "^4.3.2",
     "functional-red-black-tree": "^1.0.1",
-    "regexpp": "^3.1.0",
+    "regexpp": "^3.2.0",
     "semver": "^7.3.5",
     "tsutils": "^3.21.0"
   },
   "devDependencies": {
-    "@types/debug": "*",
-    "@types/marked": "*",
+    "@types/debug": "^4.1.7",
+    "@types/marked": "^2.0.5",
```

Comment:
quick drive-by comment
We annotate dev dependencies with `*` when the dep is declared at the root level.
The `*` allows us to avoid syncing the version numbers across multiple files.

Reply:
Keeping `@types/marked` at `*` makes it have v3.x, which has breaking type changes

It's only used in this package, so I can remove it from the root `package.json` instead if that's preferred?

Reply:
There was a reason that all the `@types/` packages were listed in the root.. but TBH I forget what it was exactly.......

---

The `yarn.lock` will ensure that `*` matches the "correct" version.
EG in the current file this listing pins the version `*` version:

https://github.com/typescript-eslint/typescript-eslint/blob/4a88de2c621ae645c070d995768006df8cb7e4df/yarn.lock#L2180-L2183

You can manually edit the yarn.lock to re-fix it afterwards if you're bumping the versions.

I've done that before a number of times - deleted the yarn.lock, run a fresh install to force upgrade everything and then tweaked the file afterward. Also had to manually edit the yarn.lock to stop older versions being installed.. the joys of non-deterministic installation algorithms.

Reply:
Will revert these changes, delete `yarn.lock`, do a fresh install and edit `yarn.lock` manually then 👍